comp/core/workloadmeta/def/types.go

```diff
@@ -1349,3 +1350,57 @@ func (e EventBundle) Acknowledge() {
 // InitHelper this should be provided as a helper to allow passing the component into
 // the inithook for additional start-time configutation.
 type InitHelper func(context.Context, Component, config.Component) error
+
+// GPU represents a GPU resource.
+type GPU struct {
+	EntityID
+	EntityMeta
+	Vendor     string
+	Model      string
```

Comment:
what is model in this case? is it device (as per the agreed tags names)?

Reply:
+1 What is the difference between model and device? Could we leave some more comments alongside the field names in the struct to help guide us and reduce ambiguity. Thanks!

Reply:
We investigated a bit to see the exact terminology that NVIDIA uses and the tags we're having for GPUs. I've changed this to `Device`, which is the commercial name of the hardware. Added also a comment clarifying.